src/signing/stronghold.rs

```diff
@@ -28,12 +28,12 @@ pub(crate) async fn stronghold_path(storage_path: &Path) -> crate::Result<PathBu
 #[async_trait::async_trait]
 impl super::Signer for StrongholdSigner {
     async fn get_ledger_status(&self, _is_simulator: bool) -> crate::LedgerStatus {
-        // dummy ledger status
-        crate::LedgerStatus::Connected
-    }
-
-    async fn get_ledger_opened_app(&self, _is_simulator: bool) -> crate::Result<crate::LedgerAppInfo> {
-        Err(crate::Error::NoLedgerSignerError)
+        // dummy status
+        crate::LedgerStatus {
+            connected: false,
```

Comment:
shouldn't the default dummy status be connected: true?

Reply:
I would rather say no, because it's not a ledger, but at the end it shouldn't be used at all, it's only required for the trait